apps/server/src/modules/authorization/feathers-jwt.provider.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain';
+import { FeathersServiceProvider } from '@shared/infra/feathers';
+
+@Injectable()
+export class FeathersJwtProvider {
+	constructor(private feathersServiceProvider: FeathersServiceProvider) {}
+
+	async generateJwt(userId: EntityId): Promise<string> {
+		const service = this.feathersServiceProvider.getService('accounts/supportJWT');
```

Comment:
@Metauriel suggestions for a different service to be used instead?

Reply:
resolved for now